dotCMS/src/main/java/com/dotcms/security/apps/AppsAPIImpl.java

```diff
@@ -549,52 +550,52 @@ private static String getServiceDescriptorDirectory() {
                                         serviceDescriptor.getKey())
                                 );
                     }
-                    builder.add(new ServiceDescriptorMeta(serviceDescriptor, file.getName()));
+                    builder.add(new AppDescriptorMeta(serviceDescriptor, file.getName()));
                     loadedServiceKeys.add(serviceDescriptor.getKey());
                 }
             } catch (IOException e) {
-                Logger.error(ServiceIntegrationAPIImpl.class,
+                Logger.error(AppsAPIImpl.class,
                         String.format("Error reading yml file `%s`.", fileName), e);
             }
         }
 
         return builder.build();
     }
 
-   private boolean validateServiceDescriptor(final ServiceDescriptor serviceDescriptor)
+   private boolean validateServiceDescriptor(final AppDescriptor appDescriptor)
            throws DotDataException {
-       if(UtilMethods.isNotSet(serviceDescriptor.getKey())){
+       if(UtilMethods.isNotSet(appDescriptor.getKey())){
           throw new DotDataException("The required field `key` isn't set on the incoming file.");
        }
 
-       if(serviceDescriptor.getKey().length() > 100){
+       if(appDescriptor.getKey().length() > 100){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using Literals in Conditional Statements](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)